src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/data/GerritProject.java

```diff
@@ -90,6 +92,7 @@ public GerritProject(
             String pattern,
             List<Branch> branches,
             List<Topic> topics,
+            List<Hashtag> hashtags,
```

Comment:
This is not a mandatory parameter, so it should not be in the `@DataboundConstructor`. For optional parameters you should use `@DataboundSetter`. This constructor is too big as it is for legacy reasons and should not have to grow any more unless there is a need for a new mandatory parameter.
And in that case you need to keep the old constructor signature in place but `@Deprecated` with default values sent to the new `@DataboundConstructor`.

Reply:
thanks for the info, move it to set method with @DataboundSetter